jobs/user_add/templates/pre-start.sh.erb

```diff
@@ -4,6 +4,8 @@ set -ex
 
 <% p('users').each do |user_hash| %>
   <%  user = user_hash['name']
+      sudo = user_hash['sudo'].nil? ? true : !!user_hash['sudo']
+      shell = user_hash['shell'].nil? ? 'bash' : user_hash['shell']
```

Comment:
can you make this an absolute path so it's more explicit and closer to what `usermod` traditionally needs. also allows for a bit more flexibility.